api/base/serializers.py

```diff
@@ -280,6 +299,54 @@ def to_representation(self, data):
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+    # Overrides ListSerializer which doesn't support multiple update by default
+    def update(self, instance, validated_data):
+        if len(instance) != len(validated_data):
+            raise exceptions.NotFound()
```

Comment:
All-or-nothing approach.  If my request had ten nodes that I wanted to update and one of the nodes didn't exist, the instance and validated_data lengths would differ.  I want to stop anything from being written, and tell the user at least one of their nodes wasn't found.  Similar to https://github.com/miki725/django-rest-framework-bulk/blob/master/rest_framework_bulk/drf3/serializers.py#L57